recipes/lissero/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "lissero" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a561f420d8a5854234ed9edafe594fef81f991b9fc4f5fc552e97e2a728359a6
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - lissero = lissero.run_lissero:run_lissero
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - click
+    - loguru
+    - biopython
+    - ispcr
+
+test:
+  imports:
+    - lissero
+    - lissero.scripts
+  commands:
+    - lissero --help
+
+about:
+  home: "https://github.com/MDU-PHL/lissero"
+  license: GPL-2.0
```

Comment:
Can you please check the license, I think it is GPL2, but you had GPL3

Reply:
@bgruening I'll fix it up. There is an inconsistency between setup.py and the LICENSE file. The LICENSE file is GPL2. That was the old license. I'll update the file in the repo. I'll also add it to the distro.

Reply:
This is now fixed.